modules/dreamview/frontend/src/store/route_editing_manager.js

```diff
@@ -20,7 +22,7 @@ export default class RouteEditingManager {
 
     currentDefaultRouting = 'none';
 
-    @action updateDefaultRoutingEndPoint(data) {
+  @action updateDefaultRoutingEndPoint(data) {
```

Comment:
At least make it aligned to `@observable...`

Reply:
thx